docs/plugins/rotate_asg_instances.md

```diff
@@ -9,17 +9,22 @@ After all outdated instances are shutdown successfully, it terminates them and r
 It allows gracefully shutting down each instance instead of terminating them and killing all the running processes.
 
 ## Configuration
-The plugin uses the ssh username specified in the `MOONSHOT_SSH_USER` or the `LOGNAME` environment variable for logging into the ASG instances to shutdown. The value should be the username with which you have the access to the instances. For example:
-```ruby
-export MOONSHOT_SSH_USER=abhishek.rana
-```
+The plugin uses the ssh configuration for logging into the ASG instances to shutdown. The value should be the username with which you have the access to the instances.
```

Comment:
`The value` here is unclear.  What value?   I presume ssh_config.ssh_user based on the block below but this sentence is unclear.